src/features/movies/moviesSlice.js

```diff
@@ -11,18 +11,12 @@ const moviesSlice = createSlice({
     totalNumberOfMovies: 0,
   },
   reducers: {
+    setMoviesPage:(state,{ payload : URLCurrentPage }) => {
+      state.currentPage = URLCurrentPage;
+    },
     setMoviesPageToFirst: (state) => {
       state.currentPage = 1;
     },
```

Comment:
A czy to jest jeszcze gdzieś użwane? Bo jeśli nie, to dziwne, że inne reducery zostały usunięte, a to nie.